docs/8-call-status.md

```diff
@@ -1,75 +1,34 @@
+Each call goes through a sequence of states in its lifecycle:
 
-### States
-
-Here is a handy dandy table with a brief list of all states: 
-
-- **started** : Platform has initiated the call
-- **ringing** : Caller has joined and pending for at least one more to join
-- **answered** : The user has answered your call
-- **rejected** : The call was rejected
-- **busy**: Caller is busy
-- **unanswered**: The call was canceled from the called before an answer
-- **timeout**: User did not answer your call with ringing_timer seconds
-- **failed**: The call failed to complete
-- **complete**: Platform has terminated this call after the call was successfully answered
-- **machine**: Platform detected an answering machine
-
-These states are valid for all 1:1 call combinations such as (IP-IP, IP - PSTN, PSTN - IP, etc...). Since these states are straightforward let's demonstrate how to subscribe to changes in state in an iOS app. 
-
-## 1.0 Setting up the iOS app 
-
-In order to subscribe to call events, our app will need to be able to make outgoing calls. We will subscribe to changes in the state of the outgoing call in the usual way. We will 
-
-## 2.0 Subscribing to a State Changes & Retrieval
+A call may pass from Created to Ringing to Answered to Complete but there are many different sets of sequences for states in a call's lifecycle. Below is a schematic diagram outlining a few sets. 
 
-The first step is to import the Stitch iOS SDK. 
+![Call_States_RTC.png](./Call_States_RTC.png)
 
-```Swift 
-import Stitch
-``` 
-
-The second step is to empty initialize a private instance of the `call` object so that we can use it later to set up a call. 
-
-``` 
-private var call: Call()
-``` 
-
-The third step is to subscribe to the call states on the `call` object.
-
-
-```Swift 
-call.memberState.subscribe { state in }
-``` 
-
-The fifth step is to program the User Interface to execute code after a change of state is logged.
+### States
 
- ```Swift
-call.memberState.subscribe { state in
-	switch state {
-		case .ringing(let member):
-		    break
-		case .answered(let member):
-		    break
-		case .rejected(let member):
-		    break
-		case .hangUp(let member):
-		    break
-		}
-	}
-```
+Here is list of of all eight of the call states: 
 
-## Try it out 
+- **started** : The call is created on the Nexmo platform
+- **ringing** : The destination has confirmed that the call is ringing
+- **answered** : The destination has answered the call
+- **rejected** : The call attempt was rejected by the Nexmo platform
+- **busy**: The destination is on the line with another caller
+- **unanswered**: The call was canceled from the called before an answer
+- **timeout**: The call timed out before it was answered
+- **failed**: The call attempt failed in the phone network
```

Comment:
not always phone network. might be ip ip. the call failed before reaching the destination

Reply:
🙉